src/test/datascience/interactiveWindow.vscode.common.test.ts

```diff
@@ -202,21 +202,6 @@ suite(`Interactive window execution`, async function () {
         await waitForTextOutput(cell, 'foo');
     });
 
-    test('Clear output', async function () {
-        // Test failing after using python insiders. Not getting expected
```

Comment:
We already have tests for this,
I'm taking this out as I dont think IW should focus on testing core functionality in the Kernel.
Just slows tests

Reply:
@amunger  @rebornix /cc